design-toolkit/components/src/components/dialog/dialog.test.tsx

```diff
@@ -12,42 +12,25 @@
 
 import { render, screen } from '@testing-library/react';
 import { describe, expect, it } from 'vitest';
+import { Button } from '../button';
 import { Dialog } from './';
-import type { DialogTriggerProps } from './types';
-
-function setup({
```

Comment:
Why did you get rid of the setup function? This is a pattern that I want our tests to follow